Tests/UITests/TooltipViewSpec.swift

```diff
@@ -37,84 +36,70 @@ class ToolTipViewSpec: QuickSpec {
         afterEach {
             mockServer.stop()
         }
+        func launchToolTip() {
+            app.buttons["app_launch"].tap()
+            if !iamView.exists {
+                app.buttons["open_modal_page"].tap()
+                expect(iamView.exists).toEventually(beTrue(), timeout: .seconds(2))
+            }
+        }
         describe("Tooltip View") {
             context("when clicking X button") {
                 beforeEach {
                     launchAppIfNecessary(context: "tooltip-data")
-                    if !iamView.exists {
-                        app.buttons["open_modal_page"].tap()
-                        expect(iamView.exists).toEventually(beTrue(), timeout: .seconds(2))
-                    }
+                }
+                afterEach {
+                    app = nil
```

Comment:
If we actually need to relaunch the app each time then I think it's better to modify `launchAppIfNecessary` method by removing the guard statement inside and renaming it to just `launchApp`

Reply:
and then this `afterEach` can be deleted (along with the other one on the bottom)